server/src/main/java/org/candlepin/sync/Exporter.java

```diff
@@ -435,7 +435,7 @@ private void exportContentAccessCerts(File baseDir, Consumer consumer) throws IO
             log.debug("Exporting content access certificate: " + contentAccessCert.getSerial());
             File file = new File(contentAccessCertDir.getCanonicalPath(),
                 contentAccessCert.getSerial().getId() + ".pem");
-            CertificateExporter crt = new CertificateExporter();
+            ScaCertificateExporter crt = new ScaCertificateExporter();
```

Comment:
More regarding the cert handling behavior here. If it behaved like a normal cert by default, we wouldn't need a custom writer for it.

Reply:
Done.